test/models/businessactivities/BusinessActivitiesSpec.scala

```diff
@@ -98,11 +101,46 @@ class BusinessActivitiesSpec extends PlaySpec with MockitoSugar with OneAppPerSu
     identifySuspiciousActivity = Some(DefaultIdentifySuspiciousActivity),
     whoIsYourAccountant = Some(DefaultWhoIsYourAccountant),
     taxMatters = Some(DefaultTaxMatters),
+    transactionRecordTypes = Some(DefaultTransactionRecordTypes),
     hasChanged = false,
     hasAccepted = true
   )
 
   val completeJson = Json.obj(
+    "involvedInOther" -> true,
+    "details" -> DefaultInvolvedInOtherDetails,
+    "expectedBusinessTurnover" -> "01",
+    "expectedAMLSTurnover" -> "01",
+    "businessFranchise" -> true,
+    "franchiseName" -> DefaultFranchiseName,
+    "isRecorded" -> true,
```

Comment:
Should we have a test for when this is None? Or are we confident this is always going to be set as true or false?

Reply:
Agreed, this should be tested as `None` is definitely a possibility; I'll push up a new commit which does so.

Reply:
Looking at the spec more closely, I think the `None` case will already be covered by the tests that validate a partial Json packet

https://github.com/hmrc/amls-frontend/blob/eb9ef660ac6d0a7ba89bc3d80871cf7706588ba7/test/models/businessactivities/BusinessActivitiesSpec.scala#L211